### DISCUSSION TITLE
Fatal error: Allowed memory size , bytes exhausted  solved.

### DISCUSSION THREAD

**Discussion by tridoxx** [GetDKAN/dkan#3867] [Q&A]
I am getting this error when I drush harvesting. 

Fatal error: Allowed memory size of 536870912 bytes exhausted (tried to allocate 943456088 bytes) in /app/data/vendor/guzzlehttp/psr7/src/Stream.php on line 95
 [warning] Drush command terminated abnormally

the way to solve it is to add more memory what they should do is write ini_set('memory_limit', ' '); and put the quantity of memory of you requiere. for example i requiere a 1gb
ini_set('memory_limit', '1000M');

src/site/settings.php in the final of file. 
![image](https://user-images.githubusercontent.com/22879539/200645755-4e3a770f-c5cb-4f11-801c-3fd7e014f4b2.png)

and after check memory on drupal portal  "admin/reports/status"
![image](https://user-images.githubusercontent.com/22879539/200645927-fce61dd7-c6eb-4fb2-93cc-3759b66de011.png)

and it fixed the problem

**Comment by tridoxx** [comment]
okay i update the max size inside the settings.php file of drupal on the "sites" directory

**Comment by janette** [comment]
https://acquia.my.site.com/s/article/360005311613-Increasing-the-memory-limit-for-Drush
